save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -81,16 +84,22 @@ typealias Status = Mono<ResponseEntity<HttpStatus>>
 class DownloadProjectController(
     private val configProperties: ConfigProperties,
     private val testDiscoveringService: TestDiscoveringService,
+    objectMapper: ObjectMapper,
     kotlinSerializationJsonEncoder: KotlinSerializationJsonEncoder,
     kotlinSerializationJsonDecoder: KotlinSerializationJsonDecoder,
 ) {
     private val log = LoggerFactory.getLogger(DownloadProjectController::class.java)
     private val webClientBackend = WebClient.builder().baseUrl(configProperties.backend).codecs {
         it.defaultCodecs().kotlinSerializationJsonEncoder(kotlinSerializationJsonEncoder)
         it.defaultCodecs().kotlinSerializationJsonDecoder(kotlinSerializationJsonDecoder)
-    }
-        .build()
-    private val webClientOrchestrator = WebClient.create(configProperties.orchestrator)
+    }.build()
+    private val webClientOrchestrator = WebClient.builder().baseUrl(configProperties.orchestrator).codecs {
+        it.defaultCodecs().kotlinSerializationJsonEncoder(kotlinSerializationJsonEncoder)
+        it.defaultCodecs().kotlinSerializationJsonDecoder(kotlinSerializationJsonDecoder)
+        it.defaultCodecs().multipartCodecs().encoder(Jackson2JsonEncoder(objectMapper))
```

Comment:
We have to use Jackson to serialize `Execution`, because it's not `@Serializable` and spring-boot's `KotlinSerializationJsonEncoder` claims that it can't serialize it (`canEncode` returns `false`)